skills/csharp/experimental/itsmskill/Dialogs/SkillDialogBase.cs

```diff
@@ -220,6 +220,8 @@ protected async Task<DialogTurnResult> AfterGetAuthTokenAsync(WaterfallStepConte
                 // When the token is cached we get a TokenResponse object.
                 if (sc.Result is ProviderTokenResponse providerTokenResponse)
                 {
+                    // Save TokenResponse to State
+                    state.AccessTokenResponse = providerTokenResponse.TokenResponse;
```

Comment:
Will token be used across turns? If not, please save it to turn state. It is not safe to persist token.

Reply:
I see.. It is used in task module..